lib/manager/components/deployment/DeploymentSettings.js

```diff
@@ -206,111 +205,116 @@ class DeploymentSettings extends Component<Props, State> {
 
   render () {
     const updaters = get(this.state, 'routerConfig.updaters') || []
-    const {project, editDisabled} = this.props
+    const {editDisabled, project} = this.props
     return (
-      <div key={project.lastUpdated} className='deployment-settings-panel'>
+      <div className='deployment-settings-panel' key={project.lastUpdated}>
         {/* Build config settings */}
-        <Panel header={
-          <h4>
+        <Panel>
+          <Panel.Heading><Panel.Title componentClass='h3'>
             <Button
               bsSize='xsmall'
-              onClick={this._clearBuildConfig}
-              className='pull-right'>{this.messages('clear')}
+              className='pull-right'
+              onClick={this._clearBuildConfig}>{this.messages('clear')}
             </Button>
-            <Icon type='cog' />{' '}
-            {this.messages('buildConfig.title')}
-          </h4>
-        }>
-          {this._getFields(FIELDS, this.state, 'buildConfig')}
+            <Icon type='cog' /> {this.messages('buildConfig.title')}
+          </Panel.Title></Panel.Heading>
+          <Panel.Body>
+            {this._getFields(FIELDS, this.state, 'buildConfig')}
+          </Panel.Body>
         </Panel>
         {/* Router config settings */}
-        <Panel header={
-          <h4>
+        <Panel>
+          <Panel.Heading><Panel.Title componentClass='h3'>
```

Comment:
Maybe doing this is more useful
```suggestion
          <Panel.Heading><Panel.Title componentClass='h4'>
```